src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/retry/delay/ConsumerDelayStrategy.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.companieshouse.chipsrestinterfacesconsumer.retry.delay;
+
+@FunctionalInterface
+public interface ConsumerDelayStrategy {
+
+    void throttle();
```

Comment:
I think it would be better to name the function delay(), it would be in keeping with the class name and other references to delay.

Reply:
or change the class name to reference throttle. I suppose what I'm saying is we have a mix of throttle and delay in teh app, i guess get consensus on what to use?

Reply:
I thought about this and decided to mix the two so that in the context of cric it is clear that when when talk about a delay we are also talking about a throttle and vice versa.

Reply:
I find it confusing @sroberts2 @ICoxCH  what do you guys think?

Reply:
I think we should decide on either delay or throttle and stick with it consistently.
Cchips-filing-consumer uses RETRY_THROTTLE_RATE_SECONDS=10, so my vote is to consistently use throttle throughout the application

Reply:
Alright I have renamed everything to "Throttle".